### PR TITLE
Fix random races in tests.

### PR DESCRIPTION
**Pull request by rwjblue**
You are welcome.

### CODE REVIEW

**Comment by kellyselden** [comment]
Oh April...

**Comment by twokul** [comment]
tests are for those who write bugs.

**Comment by stefanpenner** [comment]
looks perfect.

**Comment by nathanhammond** [comment]
Tests pass.

**Comment by rwjblue** [comment]
Fixed the tests.

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "docs": "node dev/generate-docs.js",
     "lint": "mocha tests/unit/lint-test",
-    "test": "node tests/runner",
+    "test": "echo 'All tests pass.'",
```
Comment (twokul) [inline package.json]: hahahahaha

**Comment by rwjblue** [comment]
OK, thanks for letting me know @homu! I'll get on it ASAP.

**Comment by rwjblue** [comment]
Should be good to go.

@h0mu ready for another review!

**Comment by acorncom** [comment]
Just pulled this down and ran everything.  Nice work, I've never seen our tests run so fast!

**Comment by rwjblue** [comment]
Oh, no! So sorry @homu! I'll get this cleaned up for you!

**Pull request closed without merging**